ui/src/actions/general.js

```diff
@@ -347,3 +347,26 @@ export function showConfirmClearQueueDialog(show = true) {
     show,
   };
 }
+
+export function sendDisplayImage(path, imgNum) {
+  return () => {
+    fetch(
+      `mxcube/api/v0.1/detector/display_image/?path=${path}&img_num=${imgNum}`,
+      {
+        method: 'GET',
+        credentials: 'include',
+        headers: {
+          Accept: 'application/json',
+          'Content-type': 'application/json',
+        },
+      },
+    )
+      .then((response) => response.json())
+      .then((data) => {
+        window.open(
```

Comment:
At the moment one needs to have `braggy` for this feature, but it could be made configurable if needed